specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -0,0 +1,5736 @@
+      },
+      "x-ms-azure-resource": true
+    },
+    "DiagnosticPolicy": {
+      "description": "A diagnostic policy.",
+      "required": [
+        "properties",
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Policy"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DiagnosticPolicyProperties",
+          "description": "Properties for diagnostic policy",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "DiagnosticLevel"
+    },
+    "DiagnosticPolicyProperties": {
+      "description": "A diagnostic policy transfer object.",
+      "required": [
+        "logLevel"
+      ],
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Description of the policy",
+          "type": "string"
+        },
+        "logLevel": {
+          "description": "The policy log level.",
+          "enum": [
+            "Verbose",
+            "Information",
+            "Warning",
+            "Error",
+            "Critical"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "logLevel",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DimensionProperties": {
+      "description": "properties for dimension",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "localized display name of the dimension to customer",
+          "type": "string"
+        },
+        "name": {
+          "description": "dimension name",
+          "type": "string"
+        }
+      }
+    },
+    "Entitlement": {
+      "description": "Entitlement data transfer object.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyDto"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/EntitlementProperties",
+          "description": "Properties on the entitlement",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "EntitlementList": {
+      "description": "List response for get Entitlements",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The Url of next result page.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Collection of items of type DataTransferObjects.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Entitlement"
+          }
+        }
+      }
+    },
+    "EntitlementProperties": {
+      "description": "Entitlement of proposal data transfer object.",
+      "required": [
+        "subjectId",
+        "resourceId"
+      ],
+      "type": "object",
+      "properties": {
```

Comment:
Its not clear from the API or the definition descriptions what this is or what someone uses it for.  Please update the descriptions to make it more clear

Reply:
updated the description of entitlement (see above)